network/src/noise/handshake.rs

```diff
@@ -448,19 +464,19 @@ mod test {
     ) {
         let mut rng = ::rand::rngs::StdRng::from_seed(TEST_SEED);
 
-        let client_private = x25519::PrivateKey::generate(&mut rng);
-        let client_public = client_private.public_key();
+        let client_key = x25519::PrivateKey::generate(&mut rng);
```

Comment:
pet peeve: to reflect `client_public_key` this should be `client_private_key`

Reply:
what's our naming convention? i personally prefer `key` + `pubkey`. here it's `key` + `public_key`. in other places it's `private` + `public` or `private` + `public_key`...

let's choose one and use it everywhere

Reply:
we're pretty much all over the place on this one. 
`key` is usually used for symmetric algorithms. 

I think private_key and public_key is better than private/public imo